CloudyCider/models/EC2Store.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  EC2Store.swift
+//  CloudyCider
+//
+//  Created by Matvii Hodovaniuk on 7/7/19.
+//  Copyright © 2019 Matvii Hodovaniuk. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+final class EC2Store: BindableObject {
+  public let didChange = PassthroughSubject<(), Never>()
+
+  public var errorMessage: String? {
+    didSet {
+      DispatchQueue.main.async {
+        self.didChange.send()
+      }
+    }
+  }
+
+  public var instances: [EC2Instance]? {
```

Comment:
this property and `errorMessage` should be removed since the array of instances is stored in `result`